src/Listeners/Impersonate.php

```diff
@@ -11,10 +11,10 @@
 
 namespace FoF\ModeratorNotes\Listeners;
 
+use Flarum\Locale\Translator;
```

Comment:
Core and bundled extensions are using the new Symfony contract, `Symfony\Contracts\Translation\TranslatorInterface`, as the implementation is closer to Symfony's. That should be used here too.